source/_docs/pantheon-yml.md

```diff
@@ -54,8 +54,15 @@ Add `php_version` to the top level of the `pantheon.yml` file to configure the P
 ```yaml
 php_version: 7.0
 ```
-If you are upgrading to PHP 7, you may need to make changes to your site's codebase for compatibility. If you use Drupal, you need to [switch to Drush version 7 or higher](https://pantheon.io/docs/drush-versions/#modifying-default-drush-version). For more information, see [Upgrade PHP Versions](/docs/php-versions).
+If you are upgrading to PHP 7, you may need to make changes to your site's codebase for compatibility. If you use Drupal, you need to [switch to Drush version 7 or higher](https://pantheon.io/docs/drush-versions/#configure-drush-version). For more information, see [Upgrade PHP Versions](/docs/php-versions).
 
+### Drush Version
+Add `drush_version` to the top level of the `pantheon.yml` file to configure the Drush version used when making calls remotely on Pantheon:
```

Comment:
Yeah it uses the same drush, I tested and confirmed as well as looked up the code: https://github.com/pantheon-systems/titan-mt/blob/master/pantheon/endpoint_jobs.py#L429-L432